### ISSUE TITLE
CheckStyle Plugin can not show source from the check result from gometalinter

### ISSUE THREAD

**Issue by JackTiger** [alecthomas/gometalinter#283]
I use gometalinter to generate the check xml, and I want to show the detail in jenkins, after I use the checkstyle plugin, I find that I can not know the error source. 

```
File             Line               Priority                Type           Category
bee.go:12    12                Normal                   -                    -

```

So How can I show the source in jenkins using checkstyle plugin  after I generate the check xml, thanks!

**Comment by alecthomas** [comment]
There is not enough information in this issue for me to debug. I don't use Jenkins, so you'll have to do some digging and let me know what's wrong.

**Comment by JackTiger** [comment]
@alecthomas 
Checkstyle XML format
gometalinter supports checkstyle compatible XML output format. It is triggered with --checkstyle flag:
gometalinter --checkstyle
Checkstyle format can be used to integrate gometalinter with Jenkins CI with the help of Checkstyle Plugin.

The problem I find that I can not know the warning source（like golint/govet/lll/unused）when I use Checkstyle Plugin. The CheckStyle Plugin can not parse the warning source from the xml generated by gometalinter.

**Comment by alecthomas** [comment]
Yes, I understand the problem, but you will need to debug this yourself, as I do not have or use the CheckStyle Plugin.

**Comment by alecthomas** [comment]
Please reopen if you obtain more information.

**Comment by JackTiger** [comment]
@alecthomas 
Hi ,alecthomas, I find the reason caused this probelm, I read the source of checkstyle plugin, ant the type in checkstyle is parsed from the source,  for example, if the source is golint.golint, and the type value is golint, so when gometalinter generates the xml format of checkstyle, the source value must add like source.souce(golint.golint), so that , the checkstyle plugin in jenkins can parse correctly.

the Checkstyle source is as below:
https://github.com/jenkinsci/checkstyle-plugin/blob/5ce1cbca9a4d492787ea5dee115cc22cec21d935/src/main/java/hudson/plugins/checkstyle/parser/CheckStyleParser.java#L112
